dotcom-rendering/src/web/components/Page.tsx

```diff
@@ -1,36 +1,44 @@
-import React, { StrictMode } from 'react';
-import { EmotionCache } from '@emotion/cache';
-import { CacheProvider, Global, css } from '@emotion/react';
+import { StrictMode } from 'react';
+import { Global, css } from '@emotion/react';
 import { focusHalo } from '@guardian/source-foundations';
+import { ArticleDesign } from '@guardian/libs';
+import { SkipTo } from './SkipTo';
+import { DecideLayout } from '../layouts/DecideLayout';
 
 type Props = {
-	children: React.ReactNode;
-	cache: EmotionCache;
+	CAPI: CAPIType;
+	NAV: NavType;
+	format: ArticleFormat;
 };
 
 /**
  * @description
- * Page is a high level wrapper for pages on Dotcom. Sets strict mode, some globals
- * and the Emotion cache
+ * Page is a high level wrapper for pages on Dotcom. Sets strict mode and some globals
  *
- * @param {ReactNode} children - What gets rendered on the page
- * @param {EmotionCache} cache - Provides the Emotion cache
+ * @param {CAPIType} CAPI - The article JSON data
+ * @param {NAVType} NAV - The article JSON data
+ * @param {ArticleFormat} format - The format model for the article
  * */
-export const Page = ({ children, cache }: Props) => {
+export const Page = ({ CAPI, NAV, format }: Props) => {
 	return (
-		<CacheProvider value={cache}>
-			<StrictMode>
-				<Global
-					styles={css`
-						/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
-						/* The not(.src...) selector is to work with Source's FocusStyleManager. */
-						*:focus {
-							${focusHalo}
-						}
-					`}
-				/>
-				{children}
-			</StrictMode>
-		</CacheProvider>
+		<StrictMode>
+			<Global
+				styles={css`
+					/* Crude but effective mechanism. Specific components may need to improve on this behaviour. */
+					/* The not(.src...) selector is to work with Source's FocusStyleManager. */
+					*:focus {
+						${focusHalo}
+					}
+				`}
+			/>
+			<SkipTo id="maincontent" label="Skip to main content" />
```

Comment:
Is there a reason that prior to this each of these items were rendered out on their own outside of the main (p)react scope? Just want to make sure this doesn't have any unintentional accessibility side-effects.

Reply:
I don't think there was a specific reason. They just needed to be rendered at a very high level and the `Page` component didn't exist at that point.

They still work in `code`

Reply:
> They still work in code

I lied. I checked again and _dead_ blogs were working but not liveblogs. Thanks for raising this code as suspicious 🙏 🙏 🙏 

I needed to move the brackets around manually to prevent prettier adjusting the way the logic worked